src/engine/input/RunOnKeyUpAttribute.cs

```diff
@@ -0,0 +1,27 @@
+using Godot;
+
+/// <summary>
+///   Attribute for a method, that gets called when the defined key is released.
+///   Can be applied multiple times.
+/// </summary>
+public class RunOnKeyUpAttribute : RunOnKeyAttribute
+{
+    public RunOnKeyUpAttribute(string godotInputName) : base(godotInputName)
```

Comment:
I don't think using `godotInputName` is necessary, it should just be `inputName` if we ever have some input name that doesn't match a Godot action, that should instead have a more complicated name.